cli/embedded-app/src/views/IframeEmbed.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div id="routerLayout">
+    <div class="explainer">
+      Showing
+      <span class="app-route">{{ frameRoute }}</span>
```

Comment:
I am considering having the `frame-router` emit a custom event like `transitioned` that indidcates what URL it's displaying internally. That information could be added here as a helpful sanity check. Something like:

`Showing /foo/bar as https://localhost:8080/clientapp/#/foo/bar`

Reply:
I think we have to add some way to test/show any events.  if you're testing your embedded app, you're going to want to see what happens when you send it messages or receive messages.

I'd almost envision a storybook style sidebar.

Reply:
I agree completely. That's part of the feature set I want to add soon. At this point though, somebody can at least click around their app and manually verify it's working, and I wanted to go ahead and ship that.